README.md

```diff
@@ -55,8 +55,8 @@ We recommend building on Ubuntu 16.04 LTS (64-bit)
     git submodule sync --recursive
     git submodule update --init --recursive
 
-**NOTE:** BitShares requires a [Boost](http://www.boost.org/) version in the range [1.57 - 1.65.1]. Versions earlier than
-1.57 or newer than 1.65.1 are NOT supported. If your system's Boost version is newer, then you will need to manually build
+**NOTE:** BitShares requires a [Boost](http://www.boost.org/) version in the range [1.57 - 1.69]. Versions earlier than
+1.57 or newer than 1.69 are NOT supported. If your system's Boost version is newer, then you will need to manually build
```

Comment:
The sentence was written when we knew that Boost-1.66 would not work. It should be rewritten in the sense of "1.57 up to 1.69 are supported. Newer versions may work too but have not been tested."